scripts/clean_results_csv.py

```diff
@@ -72,7 +100,8 @@ def main():
 
     ## Get several dataframes
     dr.get_compound_df()
-    dr.get_structure_df(resolution_csv="../data/mers_structures.csv")
+    if not args.keep_structure_source:
+        dr.get_structure_df(resolution_csv="../data/mers_structures.csv")
```

Comment:
I think we should try to avoid hardcoded paths here, since it make it depend on where you run it. Maybe we can use `__file__` variable for this
```suggestion
        dr.get_structure_df(resolution_csv=os.path.join(os.path.dirname(__file__), "..", "data", "mers_structures.csv"))
```
Or something similar.